samples/azure/azure-service-bus-msmq-bridge/sample.md

```diff
@@ -0,0 +1,78 @@
+---
+title: MSMQ to Azure Service Bus Transport Bridge
+reviewed: 2017-10-17
+component: ASB
+related:
+- transports/azure-service-bus
+- nservicebus/bridge
+---
+
+
```

Comment:
I'd add a short paragraph explaining the purpose of this sample, in particular, in what situation it is useful (e.g. you mentioned POC, when transport is not decided yet).

Reply:
@weralabaj added an opening paragraph to elaborate. Have a look. Thanks.